apps/ff-site/src/app/events/utils/getEventData.ts

```diff
@@ -41,3 +45,37 @@ function transformEventData(
     },
   }
 }
+
+function assertEndIsAfterStart(
```

Comment:
Ah should we also then rename the function? Since we're not using `assert` or `isAfter`? Maybe something more generic like `validateEventTime`, or something like that, you're better in naming than me lol